docs/user-guide/update-demo/index.md

```diff
@@ -13,11 +13,11 @@ here](https://github.com/kubernetes/kubernetes.github.io/tree/{{page.docsbranch}
 
 ### Step Zero: Prerequisites
 
-This example assumes that you have forked the repository and [turned up a Kubernetes cluster](/docs/getting-started-guides/):
+This example assumes that you have forked the docs repository and [turned up a Kubernetes cluster](/docs/getting-started-guides/):
 
 ```shell
-$ cd kubernetes
-$ ./cluster/kube-up.sh
+$ git clone https://github.com/kubernetes/kubernetes.github.io
+$ cd kubernetes.github.io

```

Comment:
We don't need the whole repo but just the files in https://github.com/kubernetes/kubernetes.github.io/tree/master/docs/user-guide/update-demo

cc @kubernetes/docs should we link to master or to a specific branch such as https://github.com/kubernetes/kubernetes.github.io/tree/release-1.4/docs/user-guide/update-demo